lib/core/crypto/dart_esr/src/encoding_options.dart

```diff
@@ -3,7 +3,6 @@
 import 'dart:convert';
```

Comment:
This can be added to Main - the eos client here is only used to do read operations from chain, it doesn't need keys, and it can be standalone. So it's nicer to not have a dependency on the main client being initialized, I think.